test/adagrams.test.js

```diff
@@ -120,7 +120,9 @@ describe("Adagrams", () => {
     });
 
     it("returns a score of 0 if given an empty input", () => {
-      throw "Complete test";
+       expectScores({
+      "":0
+      });
```

Comment:
When breaking up an object on multiple lines, we should add indentation to make it easier to read.
```suggestion
       expectScores({
        "": 0
      });
```

Or you could do it one line `expectScores({"": 0});`